doctr/__main__.py

```diff
@@ -148,7 +152,7 @@ def deploy(args, parser):
 
         changes = commit_docs(added=added, removed=removed)
         if changes:
-            if can_push:
+            if can_push and not args.no_push:
```

Comment:
So even if the option is passed, do we still need to run `setup_GitHub_push` (we can't write `can_push = args.push and setup_GitHub_push(...)` above)?